vue-components/src/components/RemoteControlledArea.js

```diff
@@ -22,20 +24,28 @@ export default {
       type: Boolean,
       default: false,
     },
+    eventThrottleMs: {
+      type: Number,
+      default: 50,
```

Comment:
targeting 20fps might be a bit low. Usually I double my target... If 25ms works for you, you should go with that.